Nitrox.Launcher/Program.cs

```diff
@@ -5,26 +5,27 @@
 using System.Runtime.CompilerServices;
 using Avalonia;
 using Avalonia.ReactiveUI;
+using NitroxModel.Helper;
 using NitroxModel.Logger;
 
 namespace Nitrox.Launcher;
 
 internal static class Program
 {
-    // Initialization code. Don't use any Avalonia, third-party APIs or any
-    // SynchronizationContext-reliant code before AppMain is called: things aren't initialized
-    // yet and stuff might break.
+    // Don't use any Avalonia, third-party APIs or any SynchronizationContext-reliant code before AppMain is called
+    // Things aren't initialized yet and stuff might break
     [STAThread]
     [MethodImpl(MethodImplOptions.NoInlining)]
     public static void Main(string[] args)
```

Comment:
This function should only set the assembly resolve events. This is because when the JIT compiles this method, it _**must**_ also load dependencies (which won't use the event handler as it's not set yet).

Hence why before it called out to `LoadAvalonia` at the end, where other init logic should go, too.